src/test/suite/fixtures/recorded/languages/elm/changeTypeRed.yml

```diff
@@ -0,0 +1,30 @@
+languageId: elm
```

Comment:
Mixed feelings on this one.  We do typically support "type" for return type, but not usually when it's part of a single type annotation

In Cursorless, "type" should select type declarations and type annotations

But I don't feel super strongly in this case, esp if this is the only way to declare a function return type?

Reply:
What do we do for Haskell?

Reply:
We don't support Haskell yet